mrmustard/lab_dev/circuits.py

```diff
@@ -75,6 +76,154 @@ def components(self) -> Sequence[CircuitComponent]:
         """
         return self._components
 
+    @property
+    def path(self) -> list[tuple[int, int]]:
+        r"""
+        A list describing the desired contraction path followed by the ``Simulator``.
+        """
+        return self._path
+
+    @path.setter
+    def path(self, value: list[tuple[int, int]]):
+        self._path = value
```

Comment:
I know we do this so that the docstring shows up in the documentation, but I'm not too fond of going anti-pythonic and adding getters and setters for no other reason. Is there an alternative way to document attributes?

Reply:
@ziofil here the reason (which will become apparent in the next PR) is that the setter will trigger the validation: When the user specifies a path, we want to make sure that it's a valid one, so that we know that the simulator won't fail

Reply:
oh ok, then here it's necessary.